packages/react-native/core/src/core/Stitch.ts

```diff
@@ -30,10 +30,14 @@ let appClients: { [key: string]: StitchAppClientImpl } = {};
  * Singleton class with static utility functions for initializing a [[StitchAppClient]].
  *
  * Typically, the [[Stitch.initializeDefaultAppClient]] method is all you need 
- * to instantiate the client:
+ * to instantiate the client (note that, unlike the other JS SDKs, this method returns
+ * a Promise):
```

Comment:
nit: maybe make it a bit more explicit which SDK isn't included in "the other JS SDKs"

"unlike in the other JS SDKs, this method returns  a Promise _in the React Native SDK_"?